content/en/news/releases/1.6.x/announcing-1.6.6/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Announcing Istio 1.6.6
+linktitle: 1.6.6
+subtitle: Patch Release
+description: Istio 1.6.6 patch release.
+publishdate: 2020-07-29
+release: 1.6.6
+aliases:
+    - /news/announcing-1.6.6
+---
+
+This release contains bug fixes to improve robustness. This release note describes
+what’s different between Istio 1.6.5 and Istio 1.6.6.
+
+{{< relnote >}}
+
+## Changes
+
+- **Optimized** `ResolveGatewayName` to greatly reduce the CPU usage and number of memory allocations.
+- **Fixed** `istioctl upgrade` so that it no longer checks remote component versions on dry run. ([Issue 24865](https://github.com/istio/istio/issues/24865))
+- **Fixed** Gateway missing error to produce a short error message even when several clusters with many gateways are present in the configuration.
```

Comment:
Not sure I understand this text. I'm guessing this is https://github.com/istio/istio/pull/25610 is maybe 
**Fixed** long log message for clusters with many gateways.

Reply:
That sounds better. Yeah, that's the same PR.

Reply:
Done. Thanks.